client/src/components/RecipeModalForm.jsx

```diff
@@ -18,13 +18,12 @@ export default function RecipeModalForm(props) {
   const [formData, setFormData] = useState(() => initialRender(mode, data));
 
   const onSubmit = (event) => {
-    const form =
-      /*event.currentTarget;*/ document.getElementById('recipe-form');
+    const form = /*event.currentTarget;*/ document.getElementById('recipe-form');

```

Comment:
Is this comment needed? If not, remove or add as a TODO.

Reply:
removed comment